packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -52,6 +52,7 @@ import { computronCounter } from './computron-counter.js';
 /** @import { BlockInfo } from '@agoric/internal/src/chain-utils.js' */
 /** @import { Mailbox, RunPolicy, SwingSetConfig } from '@agoric/swingset-vat' */
 /** @import { KVStore, BufferedKVStore } from './helpers/bufferedStorage.js' */
+/** @import { Counter } from '@opentelemetry/api/build/src/metrics/Metric' */
```

Comment:
Surprised this is not exported on the root of the package ?

```suggestion
/** @import { Counter } from '@opentelemetry/api' */
```